### DISCUSSION TITLE
Is it possible to train the network using my own dataset?

### DISCUSSION THREAD

**Discussion by VladHornai** [openai/whisper#364] [Q&A] [answered]
Hello,
I am quite new regarding ASR and I was wondering if it is possbile to use Whisper for inference but with a custom dataset.
Thanks!

**Comment by turnkit** [comment]
Would love this.

**Reply by VladHornai** [reply]
So, I guess it is not possible

**Comment by stefangrotz** [comment]
It should be possible. In #349 this notebook was mentioned fine-tuning Whisper to transcribe Japanese in Katakana only. https://colab.research.google.com/drive/1P4ClLkPmfsaKn2tBbRp0nVjGMRKR-EWz?usp=sharing

**Comment by sanchit-gandhi** [comment, marked as answer]
Check-out this blog for fine-tuning Whisper for multilingual ASR with Hugging Face Transformers: https://huggingface.co/blog/fine-tune-whisper

It provides a step-by-step guide to fine-tuning, right from data preparation to evaluation 🤗 There'a Google Colab so you can also run it as a notebook 😉

**Reply by VladHornai** [reply]
Thanks for the respone!
I saw that the used dataset was the one from common_voice. Is it possible to create my own dataset and use it instead the dataset from common_voice ? 
Thanks! :)

**Reply by sanchit-gandhi** [reply]
For sure! If you have files locally, you can follow this guide: https://huggingface.co/docs/datasets/audio_dataset

It also details how you can write a script to load audio data saved remotely.
